src/drivers/VideoDriverWrapper.cpp

```diff
@@ -211,7 +213,7 @@ bool VideoDriverWrapper::DestroyScreen()
 bool VideoDriverWrapper::hasExtension(const std::string& extension)
 {
     // Extension mit Leerzeichen gibts nich
-    if( extension.empty() || extension.find(' ') != std::string::npos)
+    if(!isOglEnabled_ || extension.empty() || extension.find(' ') != std::string::npos)
```

Comment:
wouldnt it be better to create some kind of "OpenGLVideoDriver" and wrap up all gl*-Calls to that base class?